### PR TITLE
chore: Upgrades lodash

### LINKED ISSUE #22: Upgrade Lodash
The version of lodash (4.17.5) has a few CVEs open against it.  Can we please upgrade?
https://github.com/advisories/GHSA-jf85-cpcp-j695

## I'm submitting a:

- [ ] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

### PR DESCRIPTION
**Pull request by ray-vgw**
Updates lodash to 4.17.21 to close security vulnerabilities.

Fixes #22 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](/okta/okta-oidc-middleware/blob/master/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #22 

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
Fixes the following CVEs:

https://github.com/advisories/GHSA-35jh-r3h4-6jhm
https://github.com/advisories/GHSA-jf85-cpcp-j695
https://github.com/advisories/GHSA-x5rq-j2xg-h7qm
https://github.com/advisories/GHSA-p6mc-m468-83gw

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "body-parser": "^1.18.2",
     "csurf": "^1.9.0",
     "express": "^4.16.3",
-    "lodash": "^4.17.5",
+    "lodash": "4.17.21",
```
Comment (denysoblohin-okta) [inline package.json:45]:
```suggestion
    "lodash": "^4.17.21",
```

Comment (ray-vgw) [reply package.json]: Fixed

**Comment by ray-vgw** [comment]
@denysoblohin-okta, fixed the review comment, ready to merge.

**Comment by ray-vgw** [comment]
@denysoblohin-okta is there an ETA for this merge?  Internally we are closing out critical vulnerabilities, and this library is currently an outlier in getting upstream fixes closed.

**Review by denysoblohin-okta — APPROVED** [review]

**Comment by denysoblohin-okta** [comment]
Internal ref: OKTA-449113

**Comment by ray-vgw** [comment]
@denysoblohin-okta , what is the ETA for the fix?  Over 1 month now to get a single package upgraded seems like a very long time

**Comment by ray-vgw** [comment]
Seems this was merged with #36

**Pull request closed without merging**